hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ServerCrashProcedure.java

```diff
@@ -65,6 +65,14 @@
     implements ServerProcedureInterface {
   private static final Logger LOG = LoggerFactory.getLogger(ServerCrashProcedure.class);
 
+  /**
+   * Configuration parameter to retain the region assignment during ServerCrashProcedure, see
```

Comment:
Can u add few more lines as code level comments here?  What is the pros and cons of setting this. Will be easy for some one coming in here later to understand clearly.

Reply:
Sure, will address it in next commit in this PR.